src/store/todo/todoActions.ts

```diff
@@ -2,16 +2,96 @@ import { Message } from "constants/types/Message";
 import { TodoType } from "constants/types/TodoTypes";
 import * as todosTypes from "./todoTypes";
 
-export const addTodos = (todo: TodoType): todosTypes.AddTodos => ({
-  type: todosTypes.ADD_TODOS,
-  payload: todo,
+// --------------- ADD TODO -----------
+
+export const addTodo = (
+  todo: TodoType,
+  callback: (message: string, typeMessage: Message) => void
+): todosTypes.AddTodo => ({
+  type: todosTypes.ADD_TODO,
+  payload: {
+    todo: todo,
+    callback: callback,
+  },
+});
+
+export const addTodoPending = (): todosTypes.AddTodoPending => ({
+  type: todosTypes.ADD_TODOS_PENDING,
+});
+
+export const addTodoResolved = (
+  todos: TodoType[]
+): todosTypes.AddTodoResolved => ({
+  type: todosTypes.ADD_TODOS_RESOLVED,
+  payload: todos,
+});
+
+export const addTodoRejected = (error: string): todosTypes.AddTodoRejected => ({
+  type: todosTypes.ADD_TODOS_REJECTED,
+  payload: error,
+});
+// ------------------------------------
+// ------------- UPDATE TODO-----------
```

Comment:
Probably should be // ------------- **DELETE** TODO----------- instead of // ------------- **UPDATE** TODO-----------